kind-config.yaml

```
@@ -0,0 +1,9 @@
+apiVersion: kind.x-k8s.io/v1alpha4
+kind: Cluster
+nodes:
+  - role: control-plane
+    extraPortMappings:
+      - containerPort: 30000
+        hostPort: 30000
+        listenAddress: "127.0.0.1"
+        protocol: tcp
```

Add a comment here as a breadcrumb if we change image registry stuff in the future?
```suggestion
      # e2e image registry service's NodePort
      - containerPort: 30000
        hostPort: 30000
        listenAddress: "127.0.0.1"
        protocol: tcp
```

---

Done. Thanks for catching that.